### PR TITLE
Update guide to run portal locally

### PR DESCRIPTION
**Pull request by paulineribeyre**
### Improvements
- Update the guide to run data-portal locally

### CODE REVIEW

File: docs/guide_running_portal_locally.md [hunk 1]
```diff
@@ -1,4 +1,19 @@
-# Guide to running portal locally with a custom portal config
+# Guide to running data-portal locally
+
+This guide describes how to test data-portal code changes or configuration changes locally.
+
+## Instructions for internal or external use
+
+1. Your Data Commons should be running and accessible, with a running `data-portal` pod.
+2. Clone the `data-portal` repo and `cd` to it.
+3. Run `npm i`.
```
Comment (mfshao) [inline docs/guide_running_portal_locally.md:9]: should we advocate on using `npm ci` in here?

**Review by mfshao — DISMISSED** [review]
one question, overall lgtm

File: docs/guide_running_portal_locally.md (hunk 1 above)
Comment (paulineribeyre) [reply docs/guide_running_portal_locally.md]: i updated that everywhere i found `npm install`

**Review by mfshao — APPROVED** [review]
thanks 🎉

**Pull request merged**